neuralpp/symbolic/context_simplifier.py

```diff
@@ -8,26 +8,12 @@
 from .parameters import sympy_evaluate
 
 
-def _is_known_to_imply(context: Context, expression: Expression) -> bool:
```

Comment:
moved this function to be a method of `Context`, so a calling statement would be `context.is_known_to_imply(expression)`, which I believe is more intuitive.